sample-tracking-status-overview-domain/src/main/groovy/life/qbic/business/projectoverview/Project.groovy

```diff
@@ -0,0 +1,63 @@
+package life.qbic.business.projectoverview
+
+import life.qbic.datamodel.samples.Sample
+
+/**
+ * <h1><short description></h1>
+ *
+ * <p><detailed description></p>
+ *
+ * @since 1.0.0
+ *
+*/
+class Project {
+
+    final String projectCode
+    final String projectDescription
+    final List<Sample> samples
+    float progress
```

Comment:
What's the intend behind the progress field? Should it indicate how many Samples are finished? If so i'd suggest a more explicit naming schema
```suggestion
    float finishedSamples
```

Reply:
No its not the amount of finished samples. That made it difficult to name the variable. I see that progress is not the best way to name the variable. I will think of a better name.